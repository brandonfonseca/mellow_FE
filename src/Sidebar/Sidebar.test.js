import React from 'react'
import {UnconnectedSidebar as Sidebar} from '.'
import { render } from '@testing-library/react'

describe('Sidebar', () => {
  it('renders the side bar', () => {
    const wrapper = render(<Sidebar/>)
    expect(wrapper).toMatchSnapshot()
  })
})
