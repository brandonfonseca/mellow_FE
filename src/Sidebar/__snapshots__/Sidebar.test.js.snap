// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar renders the side bar 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-AxjAm fshFsw"
      >
        <div
          class="sc-AxirZ bzlyMV"
        >
          mellow
        </div>
        <div
          class="sc-AxiKw eCuIQn"
        >
          <div
            class="sc-AxgMl WzAmu"
          >
            <svg>
              calendarLogo.svg
            </svg>
          </div>
          <div
            class="sc-AxhCb iYgXkR"
          >
            Daily Check-in
          </div>
          <div
            class="sc-AxhUy idTUKW"
          />
        </div>
        <div
          class="sc-AxiKw bpZzuq"
        >
          <div
            class="sc-AxgMl WzAmu"
          >
            <svg>
              graphLogo.svg
            </svg>
          </div>
          <div
            class="sc-AxhCb iYgXkR"
          >
            Dashboard
          </div>
        </div>
        <div
          class="sc-AxiKw bpZzuq"
        >
          <div
            class="sc-AxgMl WzAmu"
          >
            <svg>
              leafLogo.svg
            </svg>
          </div>
          <div
            class="sc-AxhCb iYgXkR"
          >
            Mindfulness Tools
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-AxjAm fshFsw"
    >
      <div
        class="sc-AxirZ bzlyMV"
      >
        mellow
      </div>
      <div
        class="sc-AxiKw eCuIQn"
      >
        <div
          class="sc-AxgMl WzAmu"
        >
          <svg>
            calendarLogo.svg
          </svg>
        </div>
        <div
          class="sc-AxhCb iYgXkR"
        >
          Daily Check-in
        </div>
        <div
          class="sc-AxhUy idTUKW"
        />
      </div>
      <div
        class="sc-AxiKw bpZzuq"
      >
        <div
          class="sc-AxgMl WzAmu"
        >
          <svg>
            graphLogo.svg
          </svg>
        </div>
        <div
          class="sc-AxhCb iYgXkR"
        >
          Dashboard
        </div>
      </div>
      <div
        class="sc-AxiKw bpZzuq"
      >
        <div
          class="sc-AxgMl WzAmu"
        >
          <svg>
            leafLogo.svg
          </svg>
        </div>
        <div
          class="sc-AxhCb iYgXkR"
        >
          Mindfulness Tools
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
